src/components/button/button.css

```diff
@@ -0,0 +1,105 @@
+@import url('/src/components/utils.css');
+
+.container-row{
+    justify-content: space-evenly;
```

Comment:
It should also have flex direction row as per your name

Reply:
its default so did not mentioned that, anyway i will do that

Reply:
I know that. But many time you will use this to override pre written css and there it will not work